guide/14-deep-learning/Labeling text using Doccano.ipynb

```diff
@@ -0,0 +1,82 @@
+{
```

Comment:
<p>This section starts very abruptly. We need an introduction describing what we need to do and how Docanno can be used for labelling unstructured text.</p><p> </p><p>The doccano deplyment folder needs to be linked to the GitHub repo folder. Else, people wont know how to navigate there. They need to be told to clone the arcgis-python-api repo first and this step is missing.</p><p> </p><p>Change "Once the command prompt closes, you should have Doccano running on your local system." to " Once the install script completes, you should have Doccano running on your local system."</p><p> </p><p>URL should have <strong>forward slashes not backslashes - <em>http://localhost:8000/</em></strong></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/644/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Done, please review.